docs/life_of_a_godwoken_transaction.md

```diff
@@ -7,48 +7,51 @@ Godwoken is a layer 2 rollup framework for [Nervos CKB](https://github.com/nervo
+Fundamentally, Godwoken is a layer 2 solution for layer 1 Nervos CKB blockchain. [Here](https://pudge.explorer.nervos.org/address/ckt1qpuljza4azfdsrwjzdpea6442yfqadqhv7yzfu5zknlmtusm45hpuqgqcfn63wfumts4lvrryhc35u43q3aaf5euqq6zeas9) shows one deployment of Godwoken on CKB's testnet. New layer 2 Godwoken blocks are submitted to layer 1 CKB in **block submission transactions**. The current state of the layer 2 Godwoken blockchain is committed in the cell data part of the CKB cell generated by the block submission transaction. [This transaction](https://pudge.explorer.nervos.org/transaction/0x9fd03b09d27326aa54dc53e11d7f7c01f27e570bdc645233b91664323cac0568) is an example of a block submission transaction. The chain state, as of this layer 2 block, is:
 
 ```json
-0xdab9fff9ede40ee4c4f77fea4af996e1b12cc0c4520e36eec11ecc0097a96baf33000000e1a06940a4a5f105ed0ef1c6d8bf5878c793aa62904bd1cb75431186587a57488b420000000000000000000000000000000000000000000000000000000000000000000000000000a23e00000000000000
+0xa0cf6037bfc238b179b74a30a9b12e15a4fbdd8881aebc8e5a66a8b5b5c95f0a4ca62a4c652697c96b7e02eea3a1bc21c5369bcb30626b9aee1cbb1babea11ff4f16000055a255f47f22b6f8ed1a025f971d60528f5407b7c71745c8c493a51a631bb4bfab6f000000000000000000000000000000000000000000000000000000000000000000000000000047ee34be61c51e026838c5348ab0ca3566eed50dc9b3dc676fe16a7f7d72f80fcfe4f3d680010000466f0000000000000001
 ```
 
 This is actually a `GlobalState` data structure serialized in [molecule](https://github.com/nervosnetwork/molecule)'s format:
 
 ```
 struct GlobalState {
+    rollup_config_hash: Byte32,
     account: AccountMerkleState,
     block: BlockMerkleState,
     reverted_block_root: Byte32,
+    tip_block_hash: Byte32,
+    tip_block_timestamp: Uint64,
     last_finalized_block_number: Uint64,
     // 0: running, 1: halting
     status: byte,
+    version: byte,
 }
 ```
 
-Please refer to [this file](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L18) for more definitions.
+Please refer to [this file](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol) for more definitions.
 
-The actual layer 2 Godwoken block, is also contained in the layer 1 CKB transaction. The first witness of the above mentioned transaction, contains the following data:
+The actual layer 2 Godwoken block, is also contained in the block submission transaction. The first witness of the above mentioned transaction, contains the following data:
```

Comment:
The block submission transaction also contains the actual layer 2 block. The first witness of the transaction contains the following data: